python/tvm/hybrid/calls.py

```diff
@@ -0,0 +1,92 @@
+"""Intrinsics of TVM-Python Hybrid Script for Python compilation time
+semantic support."""
+
+import ast
+from .. import api as _api
+from .. import expr as _expr
+from .. import make as _make
+from ..container import Array
+from ..ir_pass import Equal
+from ..stmt import For
+from .util import _internal_assert
+
+#pylint: disable=redefined-builtin
+
+LOOP_INTRIN = {
+    'range'    : For.Serial,
+    'unroll'   : For.Unrolled,
+    'parallel' : For.Parallel,
+    'vectorize': For.Vectorized,
+}
+
+def _range(func_id, args):
```

Comment:
func_id is not clear here, it's better to have like a name about loop annotation, or loop type.